src/v2/Apps/ArtistSeries/Components/ArtistSeriesHeader.tsx

```diff
@@ -164,13 +160,43 @@ const ArtistSeriesHeaderSmall: React.FC<ArtistSeriesHeaderProps> = props => {
   )
 }
 
+const StyledGrid = styled(Grid)`
+  @media (max-width: ${theme.flexboxgrid.breakpoints.lg - 1}px) {
+    max-width: 100%;
+  }
+`
+
+export const HeaderImage = styled(Image)`
+  border-radius: 2px;
+
+  @media (max-width: ${theme.flexboxgrid.breakpoints.sm - 1}px) {
```

Comment:
Oh, I'm not familiar with this `theme.flexboxgrid` object — is it preferable now to use this instead of the breakpoints from [Palette's theme object](https://github.com/artsy/palette/blob/master/packages/palette/src/Theme.tsx)?

Reply:
I used `theme.flexboxgrid` because it was used elsewhere in v2 Force pages -- I didn't realize Palette had a similar theme object, but that seems more reasonable.